docs-sources/includes/_Modules.md

```diff
@@ -147,6 +147,87 @@ You may notice that the content we send is in line-separated JSON format, and
 that the first thing we send is an empty array; this array, under normal
 circumstances, would contain credentials and other metadata.
 
+## Login
+
+> lib/modules/players/index.js
+
+```javascript
+var mage = require('mage');
+
+exports.register = function (state, username, password, callback) {
+  var options = {
+    acl: ['user']
+  };
+
+  mage.auth.register(state, username, password, options,  function (error, userId) {
+    if (error) {
+      return callback(error);
+    }
+
+    mage.logger.debug('Logging in as', userId);
+
+    mage.auth.login(state, username, password, callback);
+  });
+};
+```
+
+Now that we can register users, we may want to automatically log in the user by calling
+`mage.auth.login`. While the registration has not been completed in the database, our `state`
+transaction contains the information for the newly registered user, which will allow for
+`login` to complete successfully.
+
+For more information about the `auth` module, please refer to the
+[API documentation](https://mage.github.io/mage/api/classes/mage.html#auth).
+
+## ACLs
+
+> lib/modules/players/usercommands/notAccessibleToNormalUers.js
+
+```javascript
+var mage = require('mage');
+
+// Who can access this API?
+exports.acl = ['special'];
+
+// [...]
+```
+
+As you may have noticed, `mage.auth.register` receives and `acl` option allowing
+to attach to a give user different access rights. User commands can then in return
+list what ACL groups are allowed to acces that user command.
+
+For instance, in the example above, a user registered only with the `user` credential
+would not be allowed to execute this user command. Only if the 'user' or '*' wildcard ACL
+were added to the `exports.acl` array would a normal user be able to execute it.
+
+By standard, `user` is assigned to any registered players, but you may create your own ACL
```

Comment:
players -> player

Don't wanna be too nitpicky, all documentation added is a plus. But feel free to update if you're so inclined :)